src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.ts

```diff
@@ -39,6 +39,7 @@ export class ContactInfoComponent implements OnInit {
   public conflict: boolean = false;
   public form: FormGroup;
   public ContactType = DisputeContactTypeCd;
+  public DispStatus = DisputeStatus;
```

Comment:
just comment: Using "DisputeStatus" or "Status" will be easier to understand, current is like half-half, a little bit confuse

Reply:
DisputeStatus is the same name as the class, that was problematic I found previously, and it didn't like just using status maybe that is a reserved word or something.